pom.xml

```diff
@@ -44,6 +44,7 @@
         <che.docs.version>5.4.0-SNAPSHOT</che.docs.version>
         <che.lib.version>5.4.0-SNAPSHOT</che.lib.version>
         <che.version>5.4.0-SNAPSHOT</che.version>
+        <lsp4j.version>0.2.0-SNAPSHOT</lsp4j.version>
```

Comment:
lsp4j property should be moved to https://github.com/eclipse/che-dependencies BTW we need tag